R/parse.r

```diff
@@ -791,7 +791,7 @@ fast_strptime <- function(x, format, tz = "UTC", lt = TRUE, cutoff_2000 = 68L) {
                            "%z"},
                      stop("Unrecognised capture detected; please report this bug"))
 
-      str_sub(fmt, zpos, zpos + attr(zpos, "match.length") - 1) <- repl
+      fmt <- str_sub_lubridate(fmt, zpos, zpos + attr(zpos, "match.length") - 1, repl )
```

Comment:
The convention is to name it with a dot `.str_sub`. No need for lubridate because it's clear enough that it's internal function of lubridate.

Reply:
I changed it anyway while I was in there.